### ISSUE TITLE
hentry

### ISSUE THREAD

**Issue by dshanske** [pfefferle/wordpress-uf2#14]
Many themes use hentry for styling. Don't think we can take it out without messing things up.

**Comment by peterwilsoncc** [comment]
I would agree. uf2 would need to be in addition to any built in microformats.

Officially Google doesn't support uf2 so the duel markup will help there too.

**Comment by pfefferle** [comment]
I am not actively using this plugin, so feel free to submit a merge request.

**Comment by pfefferle** [comment]
but changing it, will disable the `p-comment` cascading of microformats2

**Comment by voxpelli** [comment]
+1, the `h-entry` should instead preferably be added to the `hentry` if possible, see #17. That may cause other issues, make it hard for other data to be included in the `h-entry`, but avoid the `hentry` from being parsed while still having it on the page.

**Comment by miklb** [comment]
is this a valid discussion? Seems we use h-entry and parsers are picking it up.

**Comment by pfefferle** [comment]
The parsers had issues with microformats1 and microformats2 on different elements, if backwards compatibility was enabled... not sure if this changed...

**Comment by miklb** [comment]
it hasn't been brought up in viewing my current site running 2016 & uf2 plugin is why I ask.

**Comment by pfefferle** [comment]
it is only a problem if theme authors use hentry for styling

**Comment by pfefferle** [comment]
this seems to be the same issue: https://github.com/indieweb/wordpress-uf2/issues/17

**Comment by miklb** [comment]
The question is, a lot of these discussions are over a year old, some started 3 years ago. I am basing feedback on current parsers in a real world environment. As to a theme using a styling hook, I didn't think we were removing `hentry` only adding `h-entry` am I mistaken?

**Comment by pfefferle** [comment]
Yes, I remove hentry from the post-class and add it to the body-class

**Comment by miklb** [comment]
you might in your themes but not in this plugin. I'm looking at my home page right now and the `article` has `hentry` and `h-entry`

**Comment by pfefferle** [comment]
This is only on single pages

**Comment by pfefferle** [comment]
We also have another problem... This https://github.com/indieweb/wordpress-uf2/blob/master/includes/class-uf2-post.php#L53 sets the hfeed to the body class, but hfeed also hardcoded in the HTML source, so we currently habe 2 `hfeed`s and 1 `h-feed` on the archive pages.
